<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="HomeResNegocioTenant-20240514165916" tests="1" file=".py" time="142.743" timestamp="2024-05-14T17:01:39" failures="0" errors="1" skipped="0">
	<testcase classname="HomeResNegocioTenant" name="test_start_tenant" time="142.743" timestamp="2024-05-14T17:01:39" file="HomeResNegocio.py" line="18">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=124.0.6367.202)
Stacktrace:
	GetHandleVerifier [0x00007FF757FE1522+60802]
	(No symbol) [0x00007FF757F5AC22]
	(No symbol) [0x00007FF757E17CE4]
	(No symbol) [0x00007FF757DEDFDF]
	(No symbol) [0x00007FF757E91E57]
	(No symbol) [0x00007FF757EA98D1]
	(No symbol) [0x00007FF757E8A923]
	(No symbol) [0x00007FF757E58FEC]
	(No symbol) [0x00007FF757E59C21]
	GetHandleVerifier [0x00007FF7582E41BD+3217949]
	GetHandleVerifier [0x00007FF758326157+3488183]
	GetHandleVerifier [0x00007FF75831F0DF+3459391]
	GetHandleVerifier [0x00007FF75809B8E6+823622]
	(No symbol) [0x00007FF757F65FBF]
	(No symbol) [0x00007FF757F60EE4]
	(No symbol) [0x00007FF757F61072]
	(No symbol) [0x00007FF757F518C4]
	BaseThreadInitThunk [0x00007FFCF149257D+29]
	RtlUserThreadStart [0x00007FFCF2C2AA48+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\HomeResNegocio.py", line 36, in test_start_tenant
    validate_text(self.driver,element5 , text_expected)
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Elements.py", line 452, in validate_text
    elemento = WebDriverWait(driver, 10).until(
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\expected_conditions.py", line 84, in _predicate
    return driver.find_element(*locator)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 742, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 348, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=124.0.6367.202)
Stacktrace:
	GetHandleVerifier [0x00007FF757FE1522+60802]
	(No symbol) [0x00007FF757F5AC22]
	(No symbol) [0x00007FF757E17CE4]
	(No symbol) [0x00007FF757DEDFDF]
	(No symbol) [0x00007FF757E91E57]
	(No symbol) [0x00007FF757EA98D1]
	(No symbol) [0x00007FF757E8A923]
	(No symbol) [0x00007FF757E58FEC]
	(No symbol) [0x00007FF757E59C21]
	GetHandleVerifier [0x00007FF7582E41BD+3217949]
	GetHandleVerifier [0x00007FF758326157+3488183]
	GetHandleVerifier [0x00007FF75831F0DF+3459391]
	GetHandleVerifier [0x00007FF75809B8E6+823622]
	(No symbol) [0x00007FF757F65FBF]
	(No symbol) [0x00007FF757F60EE4]
	(No symbol) [0x00007FF757F61072]
	(No symbol) [0x00007FF757F518C4]
	BaseThreadInitThunk [0x00007FFCF149257D+29]
	RtlUserThreadStart [0x00007FFCF2C2AA48+40]

]]></error>
		<system-out><![CDATA[Tiempo de espera agotado. El texto por xpaht no estÃ¡ presente.
]]></system-out>
	</testcase>
</testsuite>
